test/Test/Mockery/LoggingSpec.hs

```diff
@@ -0,0 +1,32 @@
+module Test.Mockery.LoggingSpec (main, spec) where
+
+import Test.Hspec
+import Data.IORef
+import System.Logging.Facade.Types
+import System.Logging.Facade.Sink
+import System.Logging.Facade as Log
+
+import Test.Mockery.Logging
+
+main :: IO ()
+main = hspec spec
```

Comment:
With `sensei` in place I don't really need that `main` anymore.  In case you don't need it neither, let's skip it ;).